### PR TITLE
Migrate terra-title

### LINKED ISSUE #55: Migrate terra-title into repo
https://github.com/cerner/terra-title

### PR DESCRIPTION
**Pull request by bjankord**
Resolves #55 
Migrating [terra-title](http://terra-ui-staging.herokuapp.com/terra-title/) to new repo.

screen shot 2017-03-07 at 7 21 14 pm
screen shot 2017-03-07 at 7 21 22 pm
screen shot 2017-03-07 at 7 21 28 pm

### CODE REVIEW

File: packages/terra-title/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-title",
+  "main": "lib/Title.js",
```
Comment (Matt-Butler) [inline packages/terra-title/package.json]: Title.scss

File: packages/terra-title/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "react": "^15.4.2",
+    "terra-mixins": "^1.0.0"
```
Comment (Matt-Butler) [inline packages/terra-title/package.json]: do we need these js dependencies?

Comment (bjankord) [inline packages/terra-title/package.json]: [Updated](https://github.com/cerner/terra-ui/pull/91/commits/d7cb2f343edb539d80bc0c1fd134dc6a16e7900b)

File: packages/terra-title/package.json (hunk 1 above)
Comment (bjankord) [inline packages/terra-title/package.json]: [Updated](https://github.com/cerner/terra-ui/pull/91/commits/d7cb2f343edb539d80bc0c1fd134dc6a16e7900b)

File: packages/terra-title/examples/Index.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        Terra Title Vertical Center Caption
+        <div className="terra-Title--caption">Terra Title Caption</div>
+      </span>
+    </h1>
+    <br />
+    <br />
+    <h2 id="graphic">Graphic</h2>
+    <h1 className="terra-Title terra-Title--tiny terra-Title--horizontalMiddle">
+      <span className="terra-Title--graphic">
+        <i className="fa fa-cloud" />
```
Comment (zhongyn) [inline packages/terra-title/examples/Index.jsx]: is font-awesome the dev-dependency?

Comment (bjankord) [inline packages/terra-title/examples/Index.jsx]: Ah, good question. We don't have font-awesome as a dev-dependency, [we pull in the font-awesome icons via a link tag to a CSS file for font-awesome on a cdn](https://github.com/cerner/terra-ui/pull/91/commits/39a63b9e2b5a9c804cc5e2f1cee96d8f779f4c4d).

This is only temporary, once terra-icon lands in the monorepo and we update this to a react component, we'll stop referencing font-awesome icons and switch to terra-icon component.

**Pull request merged**